test/escape-case.js

```diff
@@ -0,0 +1,111 @@
+import test from 'ava';
+import avaRuleTester from 'eslint-ava-rule-tester';
+import rule from '../rules/escape-case';
+
+const ruleTester = avaRuleTester(test, {
+	env: {
+		es6: true
+	}
+});
+
+const errors = [{
+	ruleId: 'escape-case',
+	message: 'Use uppercase characters for the value of the escape sequence.'
+}];
+
+ruleTester.run('escape-case', rule, {
+	valid: [
+		'const foo = \'\\xA9\';',
```

Comment:
Instead of escaping the inner `'`, use `"` like `'const foo = "\\xA9";'`, which makes test cases easier to read.